.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    - vendor/**/*
+    - gemfiles/**/*
+
+Metrics/LineLength:
+  Max: 140
+
+# Can't do without a little table-flip there ;)
+Style/AsciiComments:
+  Exclude:
+    - 'test/scope_test.rb'
```

Comment:
Haha, that list looks a lot like it comes from googling ;p I certainly did and stumbled across [this list](https://en.wiktionary.org/wiki/Appendix:English_words_with_diacritics).

But oh well, I'll change it back so you can write more über code. ;)